src/promptflow-tools/tests/conftest.py

```diff
@@ -45,6 +45,24 @@ def serp_connection():
     return ConnectionManager().get("serp_connection")
 
 
+def verify_custom_connection(connection: CustomConnection) -> bool:
+    import urllib.request
+    from urllib.request import HTTPError
+    from urllib.error import URLError
+
+    try:
+        urllib.request.urlopen(
+            urllib.request.Request(connection.configs['endpoint_url']),
+            timeout=50)
```

Comment:
add `return True` if successful call?

Reply:
No - this would mean that there is an endpoint which isn't checking authorization requirements. I image this would be a Sev 1 ICM.

Reply:
@gjwoods I see, seems a bit hard to understand without the comments, maybe you could some comments to line 53.